### DISCUSSION TITLE
Postal web cannot bind with Public IP

### DISCUSSION THREAD

**Discussion by northboys** [postalserver/postal#1951] [Help with using Postal] [answered]
Hii, igot error when bind postal web wit my VM public IP
<img width="1035" alt="Screen Shot 2022-04-12 at 12 47 17" src="https://user-images.githubusercontent.com/38155944/162889610-579b9367-0061-4c91-98ab-48fc096f0691.png">

I am using GCP

**Comment by uhlhosting** [comment, marked as answer]
@northboys I am not sure what you mean, once the machine has an IP, the domain is binded with DNS .  
host 34.101.134.35
35.134.101.34.in-addr.arpa domain name pointer 35.134.101.34.bc.googleusercontent.com.

You should edit your PTR records in GCP, and make sure your DNS records point to that ip.

Also have you seen this?

https://cloud.google.com/compute/docs/tutorials/sending-mail

**Comment by willpower232** [comment]
That usually means the IP address you specified isn't on the server (i.e. you can see it with `ip a`) or something else is listening on that port.
